tools/docker/healthcheck.sh

```diff
@@ -6,7 +6,7 @@ PORT=$HEALTHCHECK_PORT
 if [ -z "$HEALTHCHECK_PORT" ]; then
     # check all the TCP listening sockets, filter the dragonfly process, and fetch the port.
     # For cases when dragonfly opens multiple ports, we filter with tail to choose one of them.
-    PORT=$(netstat -tlnp | grep "1/dragonfly" | grep -oE ':[0-9]+' | cut -c2- | tail -n 1)
+    PORT=$(su - dfly -c "netstat -tlnp" | grep "1/dragonfly" | grep -oE ':[0-9]+' | cut -c2- | tail -n 1)
```

Comment:
thanks! seems that if you run `su dfly -c "netstat -tlnp"` , i.e. without dash
than it won't throw any warnings.

Reply:
updated.